.env.example

```
@@ -3,6 +3,7 @@ BATCH_WITHDRAW_ADDRESS=0xA32942798112e561d4d3ffc62c552bb0E9651b66
 PROVIDER_URL=https://kovan.infura.io/v3/<secret-key>
 LBATCH_WITHDRAW_ADDRESS=0x13c5E321cdF37e3736470bbC6E8DD95015D7B9E3
 GTCR_SUBGRAPH_URL=https://api.thegraph.com/subgraphs/name/eccentricexit/light-curate-kovan-ii
+GTCR_SUBGRAPH_CHAIN_ID=100
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The GTCR_SUBGRAPH_CHAIN_ID key should go before the GTCR_SUBGRAPH_URL key

(UnorderedKey)